index.css

```diff
@@ -1,6 +1,100 @@
-header{
-    background-color: #3C3A39;
+.content{
+background: #3C3A39;
+ width: Hug(375)px;
+height: Hug(44)px;
+gap: 0px;
+opacity: 0px;
+
+}
+
+.menu-container{
+ width: Hug (375px)px;
+height: Hug (590px)px;
+top: 92px;
+padding: 120px 24px 120px 24px;
+gap: 12px;
+opacity: 0px;
+
+    background: #1C1A19;
+
+}
+.head{
+ background: #3C3A39; 
+ width: Hug(375)px;
+height: Hug(48)px;
+top: 44px;
+padding: 2px 24px 2px 12px;
+gap: 0px;
+opacity: 0px;
+
+}
+.head a{
+    margin-left: 70%;
+    margin-top: 10px;
+
+}
+.head p{
+color: #FFF5E1;
+margin-top: 10px;
+width: 375px;
```

Comment:
consider using % for the width dimensions of your layouts, for your project to be responsive